charts/ks-devops/charts/s2i/templates/_helpers.tpl

```diff
@@ -41,15 +41,3 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
-
-{{/*
-Concrete prefix of s2i image
-*/}}
-{{- define "s2i.imagePrefix" -}}
-{{- if .Values.docker.registry }}
-{{- print .Values.docker.registry "/" .Values.docker.group }}
-{{- else }}
-{{- print .Values.docker.group }}
-{{- end }}
-{{- end }}
-
```

Comment:
Why delete these lines?

Reply:
For simplicity and consistency, I've removed the `registry.group` option here and used `image.registry` as s2i image prefix :

```diff
- docker:
+ image
-   registry: ""
+   registry: "docker.io/kubespheredev"
-   group: "kubespheredev"
```

Reply:
fair enough